### ISSUE TITLE
MyPy plugin?

### ISSUE THREAD

**Issue by sobolevn** [antonagestam/abcattrs#35]
MyPy is very useful when detecting `@abtractmethod` definitions that are unfilled.
I don't think that there's a clean way to do that with just type-hints. Maybe this can be added via a plugin?

**Comment by antonagestam** [comment]
Yes, a mypy plugin would be great and I would most likely use it myself. I think it could also work as a reference implementation for suggesting standardizing `Abstract` in the standard library.
